src/scss/index.scss

```diff
@@ -1 +1,43 @@
+@import "variables";
 
+@import "../../node_modules/bootstrap/scss/bootstrap";
+
+@import "../../node_modules/bootstrap-icons/font/bootstrap-icons.css";
+
+@font-face {
+    font-family: "Rosmatica";
+    src: url(../../rosmatika-font/RosmatikaRegular-BWA45.ttf);
+}
+
+$theme-colors:
+    $purple;
+
+$primary: $pink;
+$secondary: $indigo;
+$cta: $blue;
+    
+.profile {
+    width: 120px;
+    height: 160px;
+    border-radius: 50%;
```

Comment:
Some of your custom styles here can be achieved with bootstrap classes. For example, `border-radius: 50%` can be achieved with the `rounded-4` class.